### ISSUE TITLE
Add support for PUBSUB subcommands

### ISSUE THREAD

**Issue by dgouldin** [andymccurdy/redis-py#526]
As documented here:

http://redis.io/commands/pubsub

For example:

```
>>> import redis
>>> client = redis.from_url('redis:///')
>>> client.execute_command('PUBSUB', 'NUMSUB', 'foo')
['foo', '1']
```

**Comment by Grokzen** [comment]
@andymccurdy This one can be closed as resolved, based on the merged MR #819 that implemented the pubsub subcommands.

**Comment by andymccurdy** [comment]
Thanks for catching this. Agreed, this have been resolved.
